modules/chapter1/pages/section1.adoc

```diff
@@ -1,3 +1,44 @@
 = Section 1
```

Comment:
Use descriptive titles rather than "section 1, section 2...". I assume you can use the titles suggested in the design document https://docs.google.com/document/d/1MYtVrtK88BnLb7YS-rLI39FsSlPCbDxoRN9HP75kLpk/edit#heading=h.v8fdfu8e5zsj

Reply:
Done